setup.py

```diff
@@ -67,7 +68,7 @@
     'mock',
     'flake8 ==3.6.0',
     'coveralls',
-    'path.py', 
+    'path.py',
```

Comment:
whoops, my editor :hocho: the trailing whitespace here automatically, I can revert if anyone cares that it's not related to this PR.